lunes_cms/static/js/color_unread_feedback.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Function to highlight unread feedback with bold letters
+*/
+
+
+function markUnreadFeedback () {
+    let readByEntries = document.querySelectorAll(".field-read_by");
+    if (readByEntries.length != 0) {
+        
+        readByEntries.forEach((entry)=>{
+            if (entry.textContent == "-") {
+                //entry.parentNode.style.backgroundColor = "yellow";
```

Comment:
```suggestion
```

Reply:
oh no, this line should not be muted 🙈 This marks unread feedback entries with color 😅